pyproject.toml

```diff
@@ -76,6 +77,10 @@ tutorial = [
     "notebook",
 ]
 
+imas = [
+    "imas>=4.0.0",
```

Comment:
where is this dependency used by the code?

Reply:
I see, what is the data-dictionary needed for?

Reply:
The IMAS Data Dictionary describes how IMAS data is organised.  Here though I think you want
```suggestion
    "imas-python>=2.0.0",
```